src/utils/solidity-keccak256.ts

```diff
@@ -1,3 +1,81 @@
+import { tinyBig } from "../shared/tiny-big/tiny-big";
+import { Buffer } from "buffer";
```

Comment:
Good call, I didn't think of putting a polyfill in there. I had added that explicit import just because that Node documentation you linked to recommends added the `Buffer` explicitly.